data/projects/jetbrains-ide-plugin.md

```diff
@@ -33,14 +33,14 @@ So The project is divided into two parts:
 ##### CODING PHASE 1
 
 * A working plugin with minimal functionality exists.
-* Corresponding test cases exists.
-* Corresponding documentation exists.
 
 ##### CODING PHASE 2
 
-
+* Corresponding test cases exist.
+* Corresponding documentation exists.
```

Comment:
This should stay in phase 1. We could put the general stuff in here. this part from above "Have a general mechanism of highlighting code analysis results from external tools." needs a few milestones but thats what is missing